b2share/modules/records/ext.py

```diff
@@ -72,4 +71,6 @@ def extend_default_endpoint_prefixes():
 
     def init_config(self, app):
         """Initialize configuration."""
-        pass
+        from b2share.config import B2SHARE_RECORDS_REST_ENDPOINTS
+
+        app.config['B2SHARE_RECORDS_REST_ENDPOINTS'] = B2SHARE_RECORDS_REST_ENDPOINTS
```

Comment:
This makes it sure that B2SHARE_RECORDS_REST_ENDPOINTS is read from b2share/config.py 
For some reason there was problems with loading these envars -HK